README.md

```diff
@@ -86,6 +86,27 @@ It adds only a few lightweight `require` statements to the bootstrap process
 to support all polyfills. Implementations are then loaded on-demand when
 needed during code execution.
 
+If your project requires a minimum PHP version it is advisable to add polyfills
+for lower PHP versions to the `replace` section of your `composer.json`. 
+This removes any overhead from these polyfills as they are no longer part of your project.
+The same can be done for polyfills for extensions that you require.
+
+If your project requires php 7.0, and needs the mb extension, the replace section would look
```

Comment:
I like this pull request and I was thinking about making some PR to Symfony Docs about this. However, this phrase is confusing:

```
If your project requires php 7.0, and needs the mb extension, the replace section
would look something like this.
```

But if your project requires PHP 7, the PHP constraint in composer.json wouldn't let you use it unless you have PHP 7, so you don't need the old PHP polyfills.

Moreover, these polyfills only provide some features of PHP 7, but they can't make a PHP 7 app run on a PHP 5.x server.

Same for the MB String extension. If you add it to the Composer requirements, you can't install the project unless you have that extension ... so you don't need the polyfill.

I think we miss here some comments about not adding PHP or PHP extensions constraints to composer.json or the "ignore-platform-reqs" option. What do you think @BackEndTea?

Reply:
> But if your project requires PHP 7, the PHP constraint in composer.json wouldn't let you use it unless you have PHP 7, so you don't need the old PHP polyfills.

You are right, this message is more aimed to users who use package that has a polyfill as a dependency. This means that your project also requires the polyfill. So you add it to the replace section to reduce overhead.

Reply:
As far as adding it to the symfony docs, please do, its probably a better place for that documentation then here.